zombienet_tests/0005-migrate_solo_to_para.toml

```diff
@@ -0,0 +1,44 @@
+[relaychain]
+default_image = "docker.io/paritypr/polkadot-debug:master"
+default_command = "polkadot"
+default_args = [ "-lparachain=debug" ]
+
+chain = "rococo-local"
+
+  [[relaychain.nodes]]
+  name = "alice"
+  validator = true
+
+  [[relaychain.nodes]]
+  name = "bob"
+  validator = true
+
+
+[[parachains]]
+id = 2000
+cumulus_based = true
+
+  # run the solo chain (in our case this is also already a parachain, but as it has a different genesis it will not produce any blocks.)
+  [[parachains.collators]]
+  name = "dave"
+  validator = true
+  image = "docker.io/parity/polkadot-collator:latest"
+  command = "test-parachain"
+  args = ["-lparachain=debug"]
+
+[[parachains]]
+id = 2000
+cumulus_based = true
+add_to_genesis = false
+register_para = false
+[parachains.genesis.runtime.runtime_genesis_config.sudo]
+key = "5FHneW46xGXgs5mUiveU4sbTyGBzmstUspZC92UhjJM694ty"
```

Comment:
Why do we need to override the sudo key?

Reply:
The sudo key is updated just to update some value in the spec to produce a different `genesis-state`. The original test use this fn 

```rust
		// Set some random value in the genesis state to create a different genesis hash.
		.update_storage_parachain(|| {
			sp_io::storage::set(b"test", b"test");
		})
```
but for this overriding the sudo key works for the same porpouse.

Thanks!

Reply:
Ahh okay, yeah then we should at least keep the comment or add some second chain spec.